src/mbgl/style/layers/layer_properties.hpp.ejs

```diff
@@ -25,10 +25,13 @@ struct <%- camelize(property.name) %> : <%- layoutPropertyType(property, type) %
 
 <% } -%>
 <% for (const property of paintProperties) { -%>
-<%   if (property.name === 'heatmap-color') continue; -%>
+<%   if (property.name === 'heatmap-color') { -%>
+struct <%- camelize(property.name) %> : ColorRampProperty {};
```

Comment:
is it possible to use a type alias here instead of subclassing?

Reply:
:+1: